server/src/main/java/io/druid/server/coordinator/LoadQueuePeon.java

```diff
@@ -326,6 +310,21 @@ public void run()
     }
   }
 
+  public void start()
+  {
+    ScheduledExecutors.scheduleAtFixedRate(
+            processingExecutor,
+            Period.ZERO.toStandardDuration(),
```

Comment:
This should be configurable.  It should be fine to add to the `DruidCoordinatorConfig`.